### ISSUE TITLE
Backend functionality description

### ISSUE THREAD

**Issue by danich1** [greenelab/word-lapse#10]
Code for the server will need to be implemented to return the following information in json format for the frontend. (Example json can be found [here](https://raw.githubusercontent.com/greenelab/word-lapse/demo_json/pandemic_demo.json)). In respect to the actual code to extract the information below refer to the [website demo notebook](https://github.com/greenelab/word-lapse/blob/main/server/website_workthrough.ipynb).

- [x] A listing of neighbors for a given token
- [x] A listing of changepoints if applicable
- [x] A list of frequencies for each year for a given token  i.e. pandemic in 2020 has freq of 0.0002476615
- [ ] A listing of UMAP coordinates for a token and it's neighbors across all years (blocked by me as I have to test a few more models first)

**Comment by falquaddoomi** [comment]
All but the last requirement is added in PR #15.

**Comment by falquaddoomi** [comment]
@danich1 Should this still be an open issue? Perhaps we can track the UMAP feature separately.

**Comment by danich1** [comment]
>@danich1 Should this still be an open issue? Perhaps we can track the UMAP feature separately.

You can close this issue. I'm putting UMAP on the backburner for now as now I'm under the impression its not necessary to have. If we do end up needing it, I'll open a new issue.
